src/util/cached-prop-defs.js

```diff
@@ -0,0 +1,83 @@
+//@flow
+import assign from './assign';
+import dashCase from './dash-case';
+import getAllKeys from './get-all-keys';
+
+const CACHED_PROP_DEFS:string = '____skate_propDefs';
```

Comment:
Yes it is not necessary. It is already inferred. 
It just look better if you are used to a typed language. 
It will not make any difference on the generated code since all type annotations are stripped during the build by babel-plugin-transform-flow-strip-types.